src/Raven.Server/RavenServer.cs

```diff
@@ -1143,26 +1143,56 @@ private void ListenToNewTcpConnection(TcpListener listener)
         {
             Task.Run(async () =>
             {
+                var backoffSecondsDelay = 0;
                 TcpClient tcpClient;
-                Logger tcpAuditLog = null;
-                try
-                {
-                    tcpClient = await listener.AcceptTcpClientAsync();
-                    tcpAuditLog = LoggingSource.AuditLog.IsInfoEnabled ? LoggingSource.AuditLog.GetLogger("TcpConnections", "Audit") : null;
-                }
-                catch (ObjectDisposedException)
-                {
-                    // shutting down
-                    return;
-                }
-                catch (Exception e)
+                Logger tcpAuditLog;
+
+                while (true)
                 {
-                    if (_tcpLogger.IsInfoEnabled)
+                    try
                     {
-                        _tcpLogger.Info("Failed to accept new tcp connection", e);
+                        tcpClient = await listener.AcceptTcpClientAsync();
+                        tcpAuditLog = LoggingSource.AuditLog.IsInfoEnabled ? LoggingSource.AuditLog.GetLogger("TcpConnections", "Audit") : null;
```

Comment:
do we need to create that each time?

Reply:
We will create it only once, after accepting was successful.